docker/ubuntu_debian/Dockerfile.ubuntu18_04

```diff
@@ -0,0 +1,81 @@
+ARG DOCKER_IMG="ubuntu:18.04"
+
+FROM $DOCKER_IMG as builder
+
+ARG QBDI_ARCH="X86_64"
+
+ENV USER="docker" \
+    HOME="/home/docker" \
+    PREFIX="/usr" \
+    QBDI_PLATFORM="linux"
+
+# Get latest package list, upgrade packages, install required packages
+# and cleanup to keep container as small as possible
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive \
+    apt-get install -y --no-install-recommends \
+        bash \
+        build-essential \
+        g++ \
+        g++-multilib \
+        git \
+        libstdc++-8-dev \
+        ninja-build \
+        pkg-config \
+        wget \
+        ca-certificates \
+        python3 \
+        python3-pip \
+        python3-dev && \
+    python3 -m pip install cmake && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# create a user
+RUN adduser --disabled-password --gecos '' $USER --home $HOME && \
+    chown -R $USER:$USER $HOME
```

Comment:
I don't understand why is there a `chown` here and not in the other Dockerfiles.

Reply:
A better solution can be to create the user before running apt.